examples/resources/scylladbcloud_cluster_connection/aws-tgw-vpc-attachment.tf

```diff
@@ -0,0 +1,17 @@
+# Create AWS TGW VPC Attachment Connection for the specified cluster.
+
+resource "scylladbcloud_cluster_connection" "aws_tgw" {
+	cluster_id = 1337
+	name       = "aws-tgw-test"
+	cidrlist = ["10.201.0.0/16"]
+	type = "AWS_TGW_ATTACHMENT"
+	datacenter = "AWS_US_EAST_1"
+	data = {
+		tgwid = "tgw-08461afa1119f390a"
+ 		ramarn = "arn:aws:ram:us-east-1:043400831220:resource-share/be3b0395-1782-47cb-9ae4-6d3517c6a721"
```

Comment:
I see the two attributes are aligned, and rest of them not.

Usually if you align them all, they are easier to read, since this is an example.

Reply:
fixed